insights/parsers/krb5.py

```diff
@@ -146,9 +146,6 @@ def parse_content(self, content):
                 else:
                     is_squ = True
                     squ_section_name = line.split("=")[0].strip()
-        # Set the value to the last section
```

Comment:
This lines should not be removed since the content in the last section will be lost.

Reply:
@xiangce - its already handled in the code so need of the line